packages/php/github-actions/hook-documentation/src/Documentor.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Grow\GitHubActions\HookDocumentation;
+
+use RuntimeException;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Class Documentor
+ *
+ * @since x.x.x
+ */
+class Documentor {
+
+	/** @var array */
+	protected array $args;
+
+	public function __construct( array $args = [] ) {
```

Comment:
Maybe some doc bloc to describe the $args array?

Reply:
Done in ee0b6b3.